mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+      volume = {104},
+      number = {C7},
+      year = {1999},
+      pages = {15803-15814},
+      doi = {10.1029/1999JC900081},
+      url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JC900081}
+      }
+      @article{Cavalieri2012a,
+      author={D. J. Cavalieri and C. L. Parkinson and N. DiGirolamo and A. Ivanoff},
+      journal={IEEE Geoscience and Remote Sensing Letters},
+      title={Intersensor Calibration Between F13 SSMI and F17 SSMIS for Global Sea Ice Data Records},
+      year={2012},
+      volume={9},
+      number={2},
+      pages={233-236},
+      doi={10.1109/LGRS.2011.2166754},
+      ISSN={1545-598X},
+      month={March}
+      }
+      @Article{Cavalieri2012b,
+      AUTHOR = {Cavalieri, D. J. and Parkinson, C. L.},
+      TITLE = {Arctic sea ice variability and trends, 1979--2010},
+      JOURNAL = {The Cryosphere},
+      VOLUME = {6},
+      YEAR = {2012},
+      NUMBER = {4},
+      PAGES = {881--889},
+      URL = {https://www.the-cryosphere.net/6/881/2012/},
+      DOI = {10.5194/tc-6-881-2012}
+      }
+      @article{Parkinson1999,
+      author = {Claire L. Parkinson and Donald J. Cavalieri and Per Gloersen and H. Jay Zwally and Josefino C. Comiso},
+      title = {Arctic sea ice extents, areas, and trends, 1978–1996},
+      journal = {Journal of Geophysical Research: Oceans},
+      volume = {104},
+      number = {C9},
+      year = {1999},
+      pages = {20837-20856},
+      doi = {10.1029/1999JC900082},
+      url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JC900082}
+      }
+      @Article{Parkinson2012,
+      AUTHOR = {Parkinson, C. L. and Cavalieri, D. J.},
+      TITLE = {Antarctic sea ice variability and trends, 1979--2010},
+      JOURNAL = {The Cryosphere},
+      VOLUME = {6},
+      YEAR = {2012},
+      NUMBER = {4},
+      PAGES = {871--880},
+      URL = {https://www.the-cryosphere.net/6/871/2012/},
+      DOI = {10.5194/tc-6-871-2012}
+      }
+      @article{Zwally2002,
+      author = {H. Jay Zwally and Josefino C. Comiso and Claire L. Parkinson and Donald J. Cavalieri and Per Gloersen},
+      title = {Variability of Antarctic sea ice 1979–1998},
+      journal = {Journal of Geophysical Research: Oceans},
+      volume = {107},
+      number = {C5},
+      pages = {9-1-9-19},
+      keywords = {sea ice, Antarctic, climate, passive microwave, Southern Ocean},
+      doi = {10.1029/2000JC000733},
+      url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2000JC000733},
+      year = {2002}
+      }
+    </bibtex>
+    <dataUrls>
+      - https://neptune.gsfc.nasa.gov/uploads/files/NH_IceArea_Monthly_1978-2012.txt
+      - https://neptune.gsfc.nasa.gov/uploads/files/NH_IceExt_Monthly_1978-2012.txt
+      - https://neptune.gsfc.nasa.gov/uploads/files/SH_IceArea_Monthly_1978-2012.txt
+      - https://neptune.gsfc.nasa.gov/uploads/files/SH_IceExt_Monthly_1978-2012.txt
+    </dataUrls>
+    <preprocessing>
+      (missing)
```

Comment:
@milenaveneziani, do you have a script you used to convert these text files to NetCDF?

Reply:
yes, it's a simple matlab script.